src/Component/routes.php

```diff
@@ -31,7 +33,16 @@ public static function loadComs()
         $path = '../App/Components/';
         $disk_scan = array_diff(scandir($path), ['..', '.']);
         foreach ($disk_scan as $scans) {
-            require_once '../App/Components/'.$scans.'/routes.php';
+            $configFile = route()->com . $scans.'/component.json';
```

Comment:
If the configuration file is not existed, we should do something.

Maybe we should throw the exception about that?

Reply:
no as you see like 

````php
            if (file_exists($configFile)) {
                $file = new FileHandling();
                $c = $file->open($configFile,'readOnly')->read($configFile);
                $config = json_decode($c, true);
                if ($config['status'] === true) {
                    require_once route()->com .$scans.'/routes.php';
                }
            }
```

so we can check and ignore to include that component if there is no configuration file exists instead of showing expection